vcl/CMakeLists.txt

```diff
@@ -115,8 +115,10 @@ vxl_configure_file(${CMAKE_CURRENT_LIST_DIR}/vcl_where_root_dir.h.in ${PROJECT_B
 
 include_directories(${PROJECT_BINARY_DIR})
 set( vcl_sources
-  vcl_legacy_aliases.h ${VCL_COMPILER_DETECTION_HEADER}
```

Comment:
So is the fix simply replacing ${VCL_COMPILER_DETECTION_HEADER} with the hard coded value vcl_compiler_detection.h ?

Can the variable VCL_COMPILER_DETECTION_HEADER be removed elsewhere?

Reply:
Yes, that was the fix.

We need `VCL_COMPILER_DETECTION_HEADER` in two places. CMake complains if I remove the full path from that variable.